tools/pipelines/build-docs.yml

```diff
@@ -2,14 +2,6 @@
 # Licensed under the MIT License.
 
 # build-docs pipeline
```

Comment:
Let's be sure to re-document what this pipeline does, now that its responsibilities have changed.